src/rsz/test/pin_swap1.ok

```diff
@@ -10,431 +10,1314 @@
-sky130_fd_sc_hd__o2bb2a_2 ->
-sky130_fd_sc_hd__o2bb2a_4 ->
-sky130_fd_sc_hd__o2bb2ai_1 ->
-sky130_fd_sc_hd__o2bb2ai_2 ->
-sky130_fd_sc_hd__o2bb2ai_4 ->
-sky130_fd_sc_hd__o311a_1 ->
-sky130_fd_sc_hd__o311a_2 ->
-sky130_fd_sc_hd__o311a_4 ->
-sky130_fd_sc_hd__o311ai_0 ->
-sky130_fd_sc_hd__o311ai_1 ->
-sky130_fd_sc_hd__o311ai_2 ->
-sky130_fd_sc_hd__o311ai_4 ->
-sky130_fd_sc_hd__o31a_1 ->
-sky130_fd_sc_hd__o31a_2 ->
-sky130_fd_sc_hd__o31a_4 ->
-sky130_fd_sc_hd__o31ai_1 ->
-sky130_fd_sc_hd__o31ai_2 ->
-sky130_fd_sc_hd__o31ai_4 ->
-sky130_fd_sc_hd__o32a_1 ->
-sky130_fd_sc_hd__o32a_2 ->
-sky130_fd_sc_hd__o32a_4 ->
-sky130_fd_sc_hd__o32ai_1 ->
-sky130_fd_sc_hd__o32ai_2 ->
-sky130_fd_sc_hd__o32ai_4 ->
-sky130_fd_sc_hd__o41a_1 ->
-sky130_fd_sc_hd__o41a_2 ->
-sky130_fd_sc_hd__o41a_4 ->
-sky130_fd_sc_hd__o41ai_1 ->
-sky130_fd_sc_hd__o41ai_2 ->
-sky130_fd_sc_hd__o41ai_4 ->
-sky130_fd_sc_hd__or2_0 -> A B
-sky130_fd_sc_hd__or2_1 -> A B
-sky130_fd_sc_hd__or2_2 -> A B
-sky130_fd_sc_hd__or2_4 -> A B
-sky130_fd_sc_hd__or2b_1 ->
-sky130_fd_sc_hd__or2b_2 ->
-sky130_fd_sc_hd__or2b_4 ->
-sky130_fd_sc_hd__or3_1 -> A B C
-sky130_fd_sc_hd__or3_2 -> A B C
-sky130_fd_sc_hd__or3_4 -> A B C
-sky130_fd_sc_hd__or3b_1 -> A B
-sky130_fd_sc_hd__or3b_2 -> A B
-sky130_fd_sc_hd__or3b_4 -> A B
-sky130_fd_sc_hd__or4_1 -> A B C D
-sky130_fd_sc_hd__or4_2 -> A B C D
-sky130_fd_sc_hd__or4_4 -> A B C D
-sky130_fd_sc_hd__or4b_1 -> A B C
-sky130_fd_sc_hd__or4b_2 -> A B C
-sky130_fd_sc_hd__or4b_4 -> A B C
-sky130_fd_sc_hd__or4bb_1 -> A B
-sky130_fd_sc_hd__or4bb_2 -> A B
-sky130_fd_sc_hd__or4bb_4 -> A B
-sky130_fd_sc_hd__probe_p_8 ->
-sky130_fd_sc_hd__probec_p_8 ->
-sky130_fd_sc_hd__sdfbbn_1 ->
-sky130_fd_sc_hd__sdfbbn_2 ->
-sky130_fd_sc_hd__sdfbbp_1 ->
-sky130_fd_sc_hd__sdfrbp_1 ->
-sky130_fd_sc_hd__sdfrbp_2 ->
-sky130_fd_sc_hd__sdfrtn_1 ->
-sky130_fd_sc_hd__sdfrtp_1 ->
-sky130_fd_sc_hd__sdfrtp_2 ->
-sky130_fd_sc_hd__sdfrtp_4 ->
-sky130_fd_sc_hd__sdfsbp_1 ->
-sky130_fd_sc_hd__sdfsbp_2 ->
-sky130_fd_sc_hd__sdfstp_1 ->
-sky130_fd_sc_hd__sdfstp_2 ->
-sky130_fd_sc_hd__sdfstp_4 ->
-sky130_fd_sc_hd__sdfxbp_1 ->
-sky130_fd_sc_hd__sdfxbp_2 ->
-sky130_fd_sc_hd__sdfxtp_1 ->
-sky130_fd_sc_hd__sdfxtp_2 ->
-sky130_fd_sc_hd__sdfxtp_4 ->
-sky130_fd_sc_hd__sdlclkp_1 ->
-sky130_fd_sc_hd__sdlclkp_2 ->
-sky130_fd_sc_hd__sdlclkp_4 ->
-sky130_fd_sc_hd__sedfxbp_1 ->
-sky130_fd_sc_hd__sedfxbp_2 ->
-sky130_fd_sc_hd__sedfxtp_1 ->
-sky130_fd_sc_hd__sedfxtp_2 ->
-sky130_fd_sc_hd__sedfxtp_4 ->
-sky130_fd_sc_hd__xnor2_1 ->
-sky130_fd_sc_hd__xnor2_2 ->
-sky130_fd_sc_hd__xnor2_4 ->
-sky130_fd_sc_hd__xnor3_1 ->
-sky130_fd_sc_hd__xnor3_2 ->
-sky130_fd_sc_hd__xnor3_4 ->
-sky130_fd_sc_hd__xor2_1 ->
-sky130_fd_sc_hd__xor2_2 ->
-sky130_fd_sc_hd__xor2_4 ->
-sky130_fd_sc_hd__xor3_1 ->
-sky130_fd_sc_hd__xor3_2 ->
-sky130_fd_sc_hd__xor3_4 ->
+sky130_fd_sc_hd__a2111o_1/A1 -> A2
+sky130_fd_sc_hd__a2111o_1/A2 -> A1
+sky130_fd_sc_hd__a2111o_1/B1 -> C1 D1
+sky130_fd_sc_hd__a2111o_1/C1 -> B1 D1
+sky130_fd_sc_hd__a2111o_1/D1 -> B1 C1
```

Comment:
could this be reduced to

sky130_fd_sc_hd__a2111o_1 -> [A1, A2] [B1, C1, D1] 

?

Reply:
Yeah, but we would need to implement the merging, and really doesn't represent what the underlying code is actually calculating. I think the above is simpler and actually a better test.